templates/ha.template.yaml

```diff
@@ -29,10 +29,55 @@ Parameters:
   FindAMIFunctionArn:
     Description: ARN for Lambda function used to find AMI by name
     Type: String
+  QSS3BucketName:
```

Comment:
This Parameter is standard across all the Quick Starts. Please do not change the description and other properties of this parameter. Copy it from the main template.

Reply:
Ok, updated.